xml/release-notes.xml

```diff
@@ -160,26 +160,27 @@
 
   <!-- bsc#1005323 -->
   <section xml:id="sec.123.nouveau">
-   <title>Nouveau 3D/DRI Driver Disabled by Default</title>
+   <title>Nouveau 3D/DRI Driver crashes KDE</title>
    <para>
     With &opensuseleap; &version;, the Nouveau Mesa/DRI driver for 2D/3D
-    rendering is considered experimental. Note that this does not affect the
-    support statuses of the Nouveau kernel/KMS driver or the Nouveau
-    X.org/DDX driver for 2D rendering.
+    rendering is considered experimental. The Nouveau kernel/KMS
+    driver and the Nouveau X.org/DDX driver for 2D rendering are
+    still considered to be ok.
```

Comment:
The paragraph is much clearer now, thanks -- extra proposal: `considered to be ok` -> ` considered stable`